tests/storage/guestfs.go

```diff
@@ -158,7 +140,7 @@ var _ = SIGDescribe("[rfe_id:6364]Guestfs", func() {
 		// libguestfs-test-tool verifies the setup to run libguestfs-tools
 		It("Should successfully run libguestfs-test-tool", func() {
 			pvcClaim = "pvc-verify"
-			createPVCFilesystem(pvcClaim)
+			libstorage.CreateFSPVC(pvcClaim, "500Mi")
```

Comment:
Maybe we could use a constant for the size?

Reply:
Yep I'll do that, I just pushed a second commit to tackle a different issue

Reply:
should be better now